doc/Extensions/Applications.md

```diff
@@ -162,12 +162,12 @@ that it is owned by the user running the SNMP daemon.
 mkdir -p /var/cache/librenms/
 ```
 
-4: Verify it is working by running /etc/snmp/apache-stats.py In some
-cases urlgrabber and pycurl needs to be installed, in Debian this can
-be achieved by:
+4: Verify it is working by running /etc/snmp/apache-stats.py Packages `urlgrabber` and `pycurl` for python3 needs to be
+installed. In Debian-based systems for example you can achieve this by:
 
 ```
-apt-get install python-urlgrabber python-pycurl
+apt-get install python3-pycurl 
```

Comment:
i would write it, as mentioned in text, and be compatible on debian based systems, like before

`apt-get install python3-urlgrabber python3-pycurl`

Reply:
Can someone please test on Ubuntu 20.04? I was not able to find `python3-urlgrabber` in my installation, only via pip3. So if someone can please test it would be very helpful.

Reply:
it has to be rewritten so urllib3 is used instead of old urlgrabber

Reply:
@SourceDoctor  could you please clarify what should be rewritten? The application extension, or the docs? Or both? Just to clarify, on my Ubuntu 20.04 system it seems to be working as desired. So for the sake of this PR, maybe we keep it like it is for now? And I will embark on rewriting everything to rather use the urllib3 than urlgrabber once my skillset allows for it?